app/src/main/AndroidManifest.xml

```diff
@@ -77,6 +77,12 @@
       android:parentActivityName=".ui.main.MainActivity"
       />
 
+    <activity
+      android:name=".ui.donate.DonateActivity"
+      android:label="@string/donate_title"
+      android:theme="@style/AppTheme"
```

Comment:
Why not go with `@style/AppTheme.Color` and avoid setting the background on the activity layout xml?